VideoPost/VideoPost/Helpers/AVPlayer+Convenience.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  AVPlayer+Convenience.swift
+//  VideoPost
+//
+//  Created by Shawn Gee on 5/7/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+extension AVPlayer {
+    var isPlaying: Bool {
+        return self.rate > 0
+    }
+    
+    var assetURL: URL? {
+        return (self.currentItem?.asset as? AVURLAsset)?.url
+    }
+}
```

Comment:
Nice. I always have a hard time naming these sorts of files; `Convenience` was initially, I think, used for convenience initializers, which this doesn't contain, although it does accurately describe the content of the extension (ie "*convenience* computed variables").